live-examples/js-examples/intl/intl-datetimeformat-prototype-formattoparts.js

```diff
@@ -0,0 +1,26 @@
+const date = new Date(2012, 5);
+const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+const dateTimeFormat = new Intl.DateTimeFormat('en-US', options);
+
+const parts = dateTimeFormat.formatToParts(date);
+
+console.log(parts[0].value);
+// expected output: "Friday"
+
+console.log(parts[1].value);
+// expected output: ", "
+
+console.log(parts[2].value);
+// expected output: "June"
+
+console.log(parts[3].value);
+// expected output: " "
+
+console.log(parts[4].value);
+// expected output: "1"
+
+console.log(parts[5].value);
+// expected output: ", "
+
+console.log(parts[6].value);
```

Comment:
Maybe staying away from `console.table` would be preferable since its not supported in IE.
I understand that IE doesn't support `Intl` either, but a user might find themselves in a situation where `Intl` is polyfilled but `console.table` isn't.
Maybe I'm overthinking this, would love to hear your thoughts

Reply:
Pretty everiting is not supported by IE 🙂 Anyway ok, Just _style in the console_.

The rule should be 

if string log
if object table

Reply:
So You can `console.table(parts)`